select-url.md

```diff
@@ -158,14 +158,16 @@ class CreativeSelectionByFrequencyOperation {
     // By default, return the default url (0th index).
     let index = 0;
 
-    let count = await sharedStorage.get(data.campaignId);
-    count = count ? parseInt(count) : 0;
-
-    // If under cap, return the desired ad.
-    if (count < 3) {
-      index = 1;
-      sharedStorage.set(data.campaignId, (count + 1).toString());
-    }
+    await navigator.locks.request("creation-selection-by-frequency-lock", async (lock) => {
```

Comment:
Suggest adding a comment above as to why you're creating the lock.

Reply:
Done.